tests/functional/z_cluster/cluster_expansion/test_add_capacity_lso_multiple_device_classes.py

```
@@ -0,0 +1,205 @@
+            self.pod_factory,
+            self.bucket_factory,
+            self.rgw_bucket_factory,
+        )
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        """
+        Check that the ceph health is OK after the test.
+        """
+
+        def finalizer():
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)